examples/market_making_strategies/core/market_maker.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+
+from typing import List
+from typing import Literal
+
+from frontrunner_sdk.models.cancel_order import CancelOrder
+from frontrunner_sdk.models.order import Order
+from frontrunner_sdk.sdk import FrontrunnerSDKAsync
+"""
+1. fund wallet
+2. get market id
+3. check prob
+4. send orders
+5. listen for the trade steam
+6. refresh current orders
+"""
+
+
+class MarketMaker:
+
+  def __init__(self, market_id: str, n_orders: int = 10) -> None:
+    self.async_sdk = FrontrunnerSDKAsync()
+    self.async_fr = self.async_sdk.frontrunner
+    self.async_inj = self.async_sdk.injective
+    self.wallet = self._get_wallet()
+    self.market_id = market_id
+    self.n_orders = n_orders
+
+  def _get_wallet(self):
+    return asyncio.run(self.async_sdk.wallet())
+
+  def fund_wallet(self):
+    return asyncio.run(self.async_inj.fund_wallet_from_faucet())
```

Comment:
For these operations, why not use the sync api and skip the wrapping with asyncio.run?